region/maridia/outer/Mama Turtle Room.json

```diff
@@ -955,44 +1018,87 @@
     },
     {
-      "link": [6, 3],
-      "name": "Base",
+      "link": [6, 2],
+      "name": "Tricky Speed HiJump",
       "requires": [
-        {"or": [
-          "canHeroShot",
-          "h_canUsePowerBombs",
-          "Spazer",
-          "Wave"
-        ]}
+        "HiJump",
+        "canTrickyJump",
+        "SpeedBooster",
+        "canWalljump"
+      ],
+      "clearsObstacles": ["B"],
+      "note": [
+        "Briefly run on the top ledge on either side (left is slightly easier) then jump and wall jump on the grapple block."
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Tricky Speed HiJump, Walljumpless",
+      "requires": [
+        "h_canBackIntoCorner",
+        "HiJump",
+        "canTrickyJump",
+        "SpeedBooster"
+      ],
+      "note": [
+        "Briefly run on the top right ledge then perform a last-frame jump to barely touch the item and then fall.",
+        "It is required to moon-walk or morph turnaround back against the wall (on the left side of the room, it is instead best to press against the wall and then turn around to be 1 pixel away).",
+        "A wall-jump check as Samus is close to the item can expand her hitbox and make it possible without starting against the wall, but it is much tigher."
       ]
     },
+    {
+      "link": [6, 2],
+      "name": "Springwall",
+      "requires": [
+        "canSpringwall",
+        "canResetFallSpeed"
+      ],
+      "note": "It is possible to springwall to grab the item, and then morph and reset fall speed again after entering the water suitless to land safely on the left ledge."
+    },
+    {
+      "link": [6, 3],
+      "name": "Shoot Down and Grab Right Item",
+      "requires": [
+        {"notable": "Shoot Down and Grab Right Item"}
+      ],
+      "note": [
+        "From the safe spot above the item, shoot down the wall to break the item block.",
+        "After hearing it break, jump down and collect the item."
+      ],
+      "devNote": "It is minimally easier to shoot the block with a wide beam. A Power Bomb can also be used, but is not intuitive at this difficulty."
+    },
     {
       "link": [6, 6],
       "name": "Collect Right Item from Ledge and Return",
       "collectsItems": [3],
       "requires": [
+        {"notable": "Shoot Down and Grab Right Item"},
         {"or": [
           "canPreciseWalljump",
           {"and": [
             "canTrickyJump",
             "SpaceJump"
           ]}
-        ]},
-        {"or": [
-          "canHeroShot",
-          "h_canUsePowerBombs",
-          "Spazer",
-          "Wave"
         ]}
       ],
       "note": [
-        "From the safe spot above the item, use a Power Bomb or shoot down the wall to break the item block.",
-        "Spin jump down and collect the item, then walljump back up; or carefully use SpaceJump."
-      ]
+        "From the safe spot above the item, shoot down the wall to break the item block.",
```

Comment:
seems worth mentioning the PB option in the public note for people following the spoiler map since the weapon choices aren't explicitly in the json anymore

Reply:
I was removing the PB option here and just having the normal shot always work. I didn't find using a PB to be easier/more obvious than shooting - it felt far enough away from the item that I actually wasn't initially even expecting it to work. I could add it back if that's the way most people figure out how to get it in vanilla, it just didn't seem obvious to me.

Reply:
That is how I always got the item in vanilla. :)

If you don't want to put it in the logic that's fine, I'm just suggesting having it in the notes.

Reply:
Add the text from the other strat? "Alternatively, a Power Bomb can be used from the right ledge."